README.md

```diff
@@ -52,13 +51,18 @@ Don't. We have CI.
 
 But the command to do so is `gatsby build` in the repo's top-level directory. Keep in mind that if you do build on your local and then view the static files, links between pages won't work. Better to use `gatsby develop` to see the changes you're working on.
 
-## How CI Works
+## Tools and Frameworks
 
-Forbidden magic. Basically. (Coming soon, Warren will write about it)
+* [React](https://reactjs.org/docs/hello-world.html) - JavaScript library for building user interfaces or UI components
+* [Gatsby Components](https://www.gatsbyjs.org/docs/building-with-components/) - React-based web framework
+* [Prettier](https://prettier.io/) - Used to auto-format code 
+* [Material Design Icons](https://github.com/levrik/mdi-react) - Used to insert icons (Search for icons [here](https://materialdesignicons.com/))
 
```

Comment:
Might be helpful to add GraphQL, something along the lines of "Is natively integrated with Gatsby, used to query data for our projects directory."